pkgs/cronet_http/example/android/app/build.gradle

```diff
@@ -70,7 +68,7 @@ flutter {
 }
 
 dependencies {
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation(platform("org.jetbrains.kotlin:kotlin-bom:1.8.0"))
```

Comment:
I found a reference for this issue:
https://github.com/flutter/packages/blob/347929fc60f6c296714a8757730865bcfbfb32ce/packages/local_auth/local_auth_android/android/build.gradle#L71

Could you add a similar comment with a TODO (maybe linked to a new issue that you file) e.g.
```
    // TODO(<issue number):  org.jetbrains.kotlin:kotlin-bom artifact purpose is to align kotlin stdlib and related code versions.
    // This should be removed when https://github.com/flutter/flutter/issues/125062 is fixed.
```

Reply:
done

Reply:
Could you create a bug in the package:http issue tracking and put that in the TODO(<here>) so we don't lose track of it?

Reply:
Sure! Updated